src/examples/tutorial/essentia_python_examples.ipynb

```diff
@@ -1675,6 +1675,500 @@
+      "text/plain": [
+       "<Figure size 1080x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "duration = 5  # seconds\n",
+    "audio_slice = audio_11khz[:sr * duration]\n",
+    "\n",
+    "plt.plot(audio_slice)\n",
+    "markers = np.arange(0, len(audio_slice), sr / (global_bpm / 60))\n",
+    "for marker in markers:\n",
+    "    plt.axvline(x=marker, color='red')\n",
+    "    \n",
+    "plt.title(\"Audio waveform on top of a tempo grid\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`TempoCNN` operates on audio slices of 12 seconds with an overlap of 6 seconds by default. Additionally, the algorithm outputs the local estimations along with their probabilities. The global value is computed by majority voting by default. However, this method is only recommended when a constant tempo can be assumed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "local BPM: [125. 125. 125. 125.]\n",
+      "local probabilities: [0.9679363  0.96005017 0.9681525  0.96270114]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('local BPM: {}'.format(local_bpm))\n",
+    "print('local probabilities: {}'.format(local_probs))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Given the steadiness of the audio example, the model gave very high probabilities to all the estimations.\n",
```

Comment:
"steadiness" : not sure if this is the best word. Any alternatives?

Reply:
Alternative:
You can notice how the model gives very high probabilities to all the estimations. This is because we chose an example with a very tight beat.